### PR TITLE
Upgrade and pin dependencies

### PR DESCRIPTION
**Pull request by tedpatrick**
- Pin all dependency versions
- Upgrade all dependencies
- Remove unused dependencies via https://www.npmjs.com/package/depcheck
- All tests pass

Fixes https://github.com/pyscript/pyscript/issues/432

### CODE REVIEW

File: pyscriptjs/package.json [hunk 1]
```diff
@@ -17,45 +16,41 @@
...
-    "@codemirror/commands": "^6.1.1",
-    "@codemirror/lang-python": "^6.0.0",
-    "@codemirror/language": "^6.0.0",
-    "@codemirror/state": "^6.0.0",
-    "@codemirror/theme-one-dark": "^6.0.0",
-    "@codemirror/view": "^6.3.0",
-    "codemirror": "^6.0.0",
-    "sirv-cli": "^1.0.0",
-    "svelte-fa": "^2.4.0",
-    "svelte-promisable-stores": "^0.1.3"
```
Comment (madhur-tandon) [inline pyscriptjs/package.json:59]: seems like stores aren't used anymore? Did @antocuni eliminate them all?

**Comment by JeffersGlass** [comment]
Now that font-awesome is gone, and svelte-fa is going, can probably remove the references to `svelte-fa` in the [CSS in pyrepl.ts ](https://github.com/pyscript/pyscript/blob/54cb35b60a116b9d92d6d5d9de02d702124dcbcb/pyscriptjs/src/components/pyrepl.ts#L91). Apologies, I’m on mobile and cannot test the repercussions at this moment, just remember seeing it there.

File: pyscriptjs/package.json (hunk 1 above)
Comment (tedpatrick) [reply pyscriptjs/package.json:59]: svelte is a compiler and is normally set as a dev dependency.

stores are from `import { writable } from 'svelte/store';`

There is no use of promisable stores in the PyScript project. 
`import { promisable } from 'svelte-promisable-stores';`

Comment (madhur-tandon) [reply pyscriptjs/package.json:59]: thanks, resolving...

**Comment by tedpatrick** [comment]
removed the svelte-fa css. Also tested Play button SVG, looks great.
Screen Shot 2022-10-08 at 7 52 49 PM

**Review by madhur-tandon — APPROVED** [review]

**Pull request merged**